apigrpc/apigrpc.proto

```diff
@@ -162,6 +162,22 @@ service Nakama {
     };
   }
 
+  // Authenticate a user with a Facebook Instant Game token against the server.
+  rpc AuthenticateFacebookInstantGame (api.AuthenticateFacebookInstantGameRequest) returns (api.Session) {
```

Comment:
Also add the corresponding link/unlink operations.

Reply:
Done